spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -3998,8 +4017,21 @@ def next_char(self):
         next_char = to_text_string(cursor.selectedText())
         return next_char
 
-    def in_comment(self, cursor=None):
+    def in_comment(self, cursor=None, position=None):
+        """Returns true if the given position is inside a comment
+
+        Overrides TextEditBaseWidget.in_comment
+
+        Argument priority:
+            1. cursor
+            2. position
+            3. self.textCursor()

```

Comment:
I don't understand this part of the docstring. Could you expand it so it's clearer what you mean by it?